phpBB/config/default/routing/storage.yml

```diff
@@ -0,0 +1,22 @@
+phpbb_storage_avatar_controller:
```

Comment:
do not add _controller in the route. It is not the service definition of a controller but a route name and you will use it in the code